src/vs/workbench/api/browser/mainThreadCommands.ts

```diff
@@ -98,9 +99,12 @@ export class MainThreadCommands implements MainThreadCommandsShape {
 
 // --- command doc
 
-function _generateMarkdown(description: string | Dto<ICommandHandlerDescription> | ICommandHandlerDescription): string {
+function _generateMarkdown(description: string | ILocalizedString | Dto<ICommandHandlerDescription> | ICommandHandlerDescription): string {
 	if (typeof description === 'string') {
 		return description;
+	} else if (isLocalizedString(description)) {
```

Comment:
@gregvanl said it is ok to include these descriptions in the output of this command.